src/editors/playgroundController.ts

```diff
@@ -583,10 +583,13 @@ export default class PlaygroundController {
       .get('confirmRunCopilotCode');
 
     if (!this._connectionController.isCurrentlyConnected()) {
-      // TODO(VSCODE-618): Prompt user to connect when clicked.
-      void vscode.window.showErrorMessage(connectBeforeRunningMessage);
+      const successfullyConnected =
+        await this._connectionController.changeActiveConnection();
```

Comment:
I guess the name `changeActiveConnection` is a bit weird in this specific context since there's technically no active connection.